rust/src/wallet.rs

```diff
@@ -100,7 +100,7 @@ impl Wallet {
         )?;
 
         // Create a database (using default sled type) to store wallet data
-        let db = bdk::sled::open(data_dir.clone())?;
+        let db = bdk::sled::open(data_dir.join("wallet").clone())?;
```

Comment:
Is this a breaking change for the maker?

Reply:
yes, we'd need to move the seed files. Let's not do it then for now :)

Reply:
actually no :D